test/e2e/e2e_docker_helper.go

```diff
@@ -17,8 +17,7 @@ import (
 	"github.com/docker/docker/api/types/network"
 	"github.com/docker/docker/client"
 	"github.com/docker/docker/pkg/archive"
-	"github.com/docker/docker/pkg/system"
-	. "github.com/onsi/gomega" // nolint: stylecheck
+	"github.com/docker/docker/pkg/system" // nolint: stylecheck
```

Comment:
why the `nolint` here?

Reply:
@anusha94 This comment line was already there in the codebase.